### PR TITLE
Created single video page

### LINKED ISSUE #12: create single video page and add functionalities to cards
## Overview
- Create a single video page
- Add `add to likes`, `add to playlist` & `add to watchlater` functionalities to videocards

### PR DESCRIPTION
**Pull request by kjais1720**
PR for Issue #12 ( closes #12 )
## Overview
- Created Single video page
- Created Playlist modal to add video to a playlist
- Added all the functionalities to the video card and in the single video page
## Preview
- Preview link - https://crickstreams-git-feat-create-single-video-page-7a74ad-kjais1720.vercel.app/
- Preview image
image

### CODE REVIEW

**Review by rahulcs754 — APPROVED** [review]
The code looks clean and good.
nice design of single page

File: src/pages/videoPlayer/videoPlayer.jsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  const [currentVideo, setCurrentVideo] = useState({});
+  const [ showPlaylistModal, setShowPlaylistModal ] = useState(false)
+  const {
+    resourcesApiDispatch,
+    userResources: { likes: likedVideos, watchlater },
+  } = useUserResources();
+  const {
+    ADD_TO_LIKES,
+    ADD_TO_WATCHLATER,
+    REMOVE_FROM_WATCHLATER,
```
Comment (SimranjitKaurMaan) [inline src/pages/videoPlayer/videoPlayer.jsx]: Can be `REMOVE_FROM_WATCH_LATER`

File: src/components/videoCard.jsx [hunk 2]
```diff
@@ -42,27 +42,45 @@ export function VideoCard({ video }) {
     }
   };
 
-  const addToLikesHandler = () =>
+  const addToLikes = () =>
     isLiked
       ? resourcesApiDispatch({ type: REMOVE_FROM_LIKES, payload: id })
```
Comment (SimranjitKaurMaan) [inline src/components/videoCard.jsx]: `resourcesApiDispatch` can be better named

**Review by SimranjitKaurMaan — APPROVED** [review]
Few improvements suggested. Rest looks fine.

File: src/components/videoCard.jsx (hunk 2 above)
Comment (kjais1720) [reply src/components/videoCard.jsx]: Thanks for the suggestion, I have fixed it in the latest commit

File: src/pages/videoPlayer/videoPlayer.jsx (hunk 1 above)
Comment (kjais1720) [reply src/pages/videoPlayer/videoPlayer.jsx]: Thanks for the suggestion, I have fixed it in the latest commit

**Pull request merged**